### ISSUE TITLE
Uncaught TypeError: Cannot read property '_grid' of undefined

### ISSUE THREAD

**Issue by jlchereau** [telerik/kendo-ui-core#2156]
### Bug report

This is a follow up to https://github.com/telerik/kendo-ui-core/issues/1990 because the only workaround entails another bug.

![image](https://cloud.githubusercontent.com/assets/2556751/18503212/8b04cee6-7a53-11e6-9ca4-62a641b05690.png)
### Reproduction of the problem

Use the dojo provided as the workaround at http://runner.telerik.io/fullscreen/@mishoo/inEcA
But just after clicking the **Click here to open the window** button, make sure you move the mouse.
### Current behavior

```
kendo.all.js:106326  Uncaught TypeError: Cannot read property '_grid' of undefined
    objectAt @ kendo.all.js:106326
    objectAt @ kendo.all.js:105541
    onMouseMove @ kendo.all.js:105265
    handleEvent @ kendo.all.js:95702
    mouse @ kendo.all.js:95681
    dispatch @ jquery.min.js:3
    r.handle @ jquery.min.js:3
```
### Expected/desired behavior

No error logged in the JavaScript console
### Environment

As provided in the dojo.
Reproduced with KendoUI v2016.1.714 in Chrome on Windows 10 (latest versions).

![image](https://cloud.githubusercontent.com/assets/2556751/18503384/97d1810e-7a54-11e6-8f01-f48eed713562.png)

**Comment by jlchereau** [comment]
TBC: it seems to be fixed in Kendo UI 2016.3.914

**Comment by shivangsanghi** [comment]
Hi jlchereau,
This not seems to be working in Kendo UI 2016.3.1028
